RecoLocalMuon/GEMSegment/plugins/ME0SegmentAlgorithm.cc

```diff
@@ -319,15 +318,17 @@ void ME0SegmentAlgorithm::buildSegments(const ME0Ensemble& ensemble, const Ensem
     
     muonRecHits.push_back(newRH);    
   }
+  if (rechits.size() < minHitsPerSegment) return;
```

Comment:
should i change back to the start of the function? 
I thought it would be best to just keep it the same as the previous version.